### PR TITLE
Update landing page

### PR DESCRIPTION
**Pull request by klee97**
follow up: change picture in the header to include our products

### CODE REVIEW

File: _includes/about.html [hunk 1]
```diff
@@ -1,20 +1,21 @@
...
         <div class="row">
             <div class="col-lg-8 col-lg-offset-2 text-center">
-                <h2 class="section-heading">It's hard work to collaborate on floral designs.</h2>
-                <hr class="light">
-                <p class="text-faded">A picture is worth a thousand words. But for custom flower orders, that picture won’t exist until the flower arrangement has already been made. Until then, clients and florists have to find other ways to communicate their artistic vision: emails, phone calls, Pinterest boards. It’s a process that takes trial and error, especially when a client doesn’t know what they want.
+                <h2 class="section-heading">
+                    At Foxtail, we know that the best custom bouquets come from local florist shops.</h2>
+                <!-- <hr class="light"> -->
+                <p>
+                    But as customers move online, they spend more time on your website than your shop.
```
Comment (imalao) [inline _includes/about.html]: nit: 
```suggestion
                    But as customers move online, they spend more time on your website than in your shop.
```

File: _includes/about.html [hunk 2]
```diff
@@ -1,20 +1,21 @@
...
                 </p>
-                <p class="text-left">BEFORE FOXTAIL: </p>
-                <img src="img/before.png" class="img-responsive" alt="">
-                <p class="text-faded">
-                    At Foxtail, we turn floral design ideas into visual images early on in the design process. We’re building online tools to make it easier for clients and florists to understand what an order will look like, from color palettes to shape and style.
+                <h2>
+
+                <p>                    
+                    Foxtail enhances your website by empowering your customers to build their own bouquets online.
+                    Our creative tools for custom boquets are easy to add on to your website—no technology experience required.
```
Comment (imalao) [inline _includes/about.html]:
```suggestion
                    Our creative tools for custom bouquets are easy to add to your website--no technology experience required.
```

File: _includes/about.html [hunk 3]
```diff
@@ -1,20 +1,21 @@
-<section class="bg-primary" id="about">
+<section class="bg-dark" id="about">
     <div class="container">
         <div class="row">
             <div class="col-lg-8 col-lg-offset-2 text-center">
-                <h2 class="section-heading">It's hard work to collaborate on floral designs.</h2>
-                <hr class="light">
-                <p class="text-faded">A picture is worth a thousand words. But for custom flower orders, that picture won’t exist until the flower arrangement has already been made. Until then, clients and florists have to find other ways to communicate their artistic vision: emails, phone calls, Pinterest boards. It’s a process that takes trial and error, especially when a client doesn’t know what they want.
+                <h2 class="section-heading">
+                    At Foxtail, we know that the best custom bouquets come from local florist shops.</h2>
```
Comment (imalao) [inline _includes/about.html:6]:
```suggestion
                    At Foxtail, we know that the best custom bouquets come from local florists.</h2>
```

**Review by imalao — APPROVED** [review]

File: _includes/about.html (hunk 2 above)
Comment (klee97) [reply _includes/about.html]: this is actually an em dash, so it'll appear as a long dash. fixed the other typos

File: _includes/about.html (hunk 3 above)
Comment (klee97) [reply _includes/about.html:6]: I want to include the "shop" part here (unless it sounds awkward)

**Pull request merged**